x/ethbridge/keeper/keeper.go

```diff
@@ -99,13 +99,19 @@ func (k Keeper) ProcessSuccessfulClaim(ctx sdk.Context, claim *types.EthBridgeCl
 func (k Keeper) ProcessBurn(ctx sdk.Context, cosmosSender sdk.AccAddress, msg *types.MsgBurn) error {
 	logger := k.Logger(ctx)
 	var coins sdk.Coins
+	networkIdentity := oracletypes.NewNetworkIdentity(msg.NetworkDescriptor)
+	nativeToken, err := k.oracleKeeper.GetNativeToken(ctx, networkIdentity)
```

Comment:
We should add a validation here that checks that the msg.NativeTokenSymbol is the nativeToken that we got from the keeper.

Reply:
agree, need check it.

Reply:
So we don't need to add this field into the msg because the network id maps to that symbol. That makes sense to me.